vue/src/interactions/mouseEvents.ts

```diff
@@ -96,21 +79,14 @@ const popupLogic = (map: ShallowRef<null | Map>) => {
         }
       );
       popup.setLngLat(coordinates).setHTML('<div id="popup-content"></div>').addTo(map.value);
-      const MyNewPopup = defineComponent({
-        extends: PopUpComponent,
-        setup() {
-          return { data: popupData }
-        },
-      })
       nextTick(() => {
-        if (app !== null) {
+        if (app != null) {
```

Comment:
Why change to a less strict equality check?

Reply:
that was a mistake, I'll fix it and I'm surprised the linter didn't yell at me.